app/components/log-in/log-in-form/component.js

```diff
@@ -0,0 +1,42 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { action } from "@ember/object";
+import { tracked } from "@glimmer/tracking";
+
+
+export default class LogInFormComponent extends Component {
+  @tracked errorMessage;
+  @tracked isAuthenticated = false;
+  @service store;
+  @service session;
+
+  @action
+  async authenticate(e) {
+    e.preventDefault();
+    let { identification, password } = this;
```

Comment:
Yeah, this file looks very clear 👏  Just one thing - if you not override variable, use `const` instead `let`